pkg/client/core/service_argocd_client.go

```diff
@@ -29,12 +29,13 @@ type argoCDService struct {
 
 // nolint: gosec
 const (
-	argoClientSecretName = "argocd/client_secret"
-	argoSecretKeyName    = "argocd/secret_key"
-	argoPurpose          = "argocd"
-	argoPrivateKeyName   = "argocd-privatekey"
-	argoSecretName       = "argocd-secret"
-	argoChartTimeout     = 15 * time.Minute
+	argoClientSecretName  = "argocd/client_secret"
+	argoSecretKeyName     = "argocd/secret_key"
+	argoPurpose           = "argocd"
+	argoPrivateKeyName    = "argocd-privatekey"
+	argoSecretName        = "argocd-secret"
+	argoChartTimeout      = 15 * time.Minute
```

Comment:
Did you find out what this is, compared to `constant/api.go`'s `DefaultChartApplyTimeout`?

I cannot find anywhere where argoChartTimeout is used, except maybe in the `RawYAML` function.

If you figured it out, should we add a comment explaining this somewhere? It is somewhat hard to grasp.

Reply:
https://trello.com/c/zrS1xDXz <- I tried to explain it there. I'll link to the issue in code